.github/workflows/release_upload_bundle.yml

```diff
@@ -0,0 +1,102 @@
+name: Release - Upload Bundle to Play Store
+
+on:
+  workflow_dispatch:
+
+env:
+  ASANA_PAT: ${{ secrets.GH_ASANA_SECRET }}
+  GOOGLE_APPLICATION_CREDENTIALS: '#{ENV["HOME"]}/jenkins_static/com.duckduckgo.mobile.android/ddg-upload-firebase.json'
+
+jobs:
+  create-tag:
+    name: Generate and upload nightly bundle to Play Store Internal track
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          token: ${{ secrets.GT_DAXMOBILE }}
+
+      - name: Set Git permissions
+        uses: oleksiyrudenko/gha-git-credentials@v2-latest
+        with:
+          token: '${{ secrets.GT_DAXMOBILE }}'
+
+      - name: Set up ruby env
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
+          bundler-cache: true
+
+      - name: Get Previous tag
+        id: fetch_tag
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: duckduckgo/android
+          prefix: '5.'
+
+      - name: Decode upload keys
+        uses: davidSchuppa/base64Secret-toFile-action@v2
+        with:
+          secret: ${{ secrets.UPLOAD_RELEASE_PROPERTIES }}
+          fileName: ddg_android_build_upload.properties
+          destination-path: $HOME/jenkins_static/com.duckduckgo.mobile.android/
```

Comment:
Yes. We won’t need this once Jenkins is gone. That being said, devs already have this file structure locally so changing it would involve more work like Onboarding, etc. I’ll leave this change to the very last of the migration,